core/src/main/java/org/ehcache/Ehcache.java

```diff
@@ -1064,21 +1079,25 @@ public CacheStatistics getStatistics() {
   }
 
   void create() {
+    logger.info("Cache getting created.");
```

Comment:
Would have a message like: Creating cache storage.
Cause that is really what's happening there. The Cache is created at that point.

Same holds for the other create log statements and for the destroy ones.